lib/elm-test.js

```diff
@@ -360,45 +197,52 @@ if (isMake) {
     projectElmJson
   );
 
-  function run() {
+  async function run() {
     // This compiles all the tests so that we generate *.elmi files for them,
     // which we can then read to determine which tests need to be run.
```

Comment:
```suggestion
```

This comment is out of date now but also the code is nicely structured so hopefully the purpose of the function should be clearer.